### ISSUE TITLE
Stop command

### ISSUE THREAD

**Issue by Ceddicedced** [timvisee/lazymc#72]
Really apreciate your work!

Something that is missing for me is a way to stop lazymc without ctrl+c.
That is because I'm running my servers with craftly controller and with that web interface i can just send commands to the running program. 
It would be very helpfull to have have a command that stops lazymc something like exit or quit.

**Comment by samin50** [comment]
I agree, this would be very nice to have because I also use crafty controller! Lazymc seems to be able to intepret the server 'stop' command as putting the server to sleep, so something like a 'kill' command to directly tell Lazymc to quit would be very useful.
